File: C++/Imieniny.cpp
#include <iostream>

int main()
{
    int D, L, C;
    std::cin >> D;
    while (D--)
    {
        std::cin >> L >> C;
        if (L == 1)
            std::cout << "TAK\n";
        else if (C % (L - 1) == 0)
            std::cout << "NIE\n";
        else
            std::cout << "TAK\n";
    }
    return 0;
}

File: Java/Imieniny.java
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int D = input.nextInt();
        for (int i = 0; i < D; i++) 
        {
            int L = input.nextInt();
            int C = input.nextInt();
            if (L == 1 || C % (L - 1) != 0)
                System.out.println("TAK");
            else
                System.out.println("NIE");
        }
        input.close();
    }
}
